docker-compose.yml

```diff
@@ -24,6 +24,16 @@ services:
       - db
     volumes:
       - ./volumes/nextcloud:/var/www/html
+    environment:
+      - POSTGRES_DB=db
+      - POSTGRES_HOST=${POSTGRES_HOST:-db}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-SECRET_PASSWORD}
+      - POSTGRES_USER=${POSTGRES_USER:-nextcloud}
+      - NEXTCLOUD_ADMIN_USER=${NEXTCLOUD_ADMIN_USER:-admin}
+      - NEXTCLOUD_ADMIN_PASSWORD=${NEXTCLOUD_ADMIN_PASSWORD:-admin}
+      - NEXTCLOUD_TRUSTED_DOMAINS=${NEXTCLOUD_TRUSTED_DOMAINS:-mydomain.coop}    
+    env_file:
+      - .env  
```

Comment:
The default file is `.env` and pass on docker-compose.yml normally is unnecessary. On your tests it was required specify this file here?

Reply:
I revalidated the code and saw that there is no need to declare the **.env** again. This code already delivers the DB and the "admin" user configured, according to your code for NEXTCLOUD in a local environment.
I will submit a new pull request for validation.